app/controllers/account/information.js

```diff
@@ -38,16 +38,19 @@ export default Controller.extend(ajaxStatus, {
           changes = [];
 
       for (let field in changedAttributes) {
-        if (!employeeMutableAttrs.includes(field)) {
-          return;
-        }
+        if (Object.prototype.hasOwnProperty.call(changedAttributes, field)) {
```

Comment:
Any reason for the prototype style invocation? Also just negate the logic..
```suggestion
        if (!changedAttributes.hasOwnProperty(field)) {
          continue;
        }
```